### ISSUE TITLE
Failed to set up

### ISSUE THREAD

**Issue by Daemo00** [fondberg/spotcast#427]
# Bug Ticket

## Describe the bug

Just updated Home Assistant core to 2024.3.0b0 and a notification reported this error in the logs.
Everything was working fine before the update.

## Troubleshooting

* [x] Using latest version of spotcast
* [ ] Using latest stable version of Home Assistant
* [x] I have setup the Spotify integration in Home Assistant
* [ ] I have renewed my `sp_dc` and `sp_key` values and restarted Home Assistant (see [README](https://github.com/fondberg/spotcast#enabling-debug-log))
* [ ] (optional) I have Spotify Premium
* [ ] (optional) I am using multiple accounts
* [ ] (optional) I'm attaching relevant logs with level debug for component spotcast (see [README](https://github.com/fondberg/spotcast#enabling-debug-log))
* [ ] (optional) I'm using entity_id in the service call and have tried device_name but the issue remains

## Environment

 - Installation type: [HA_OS]
 - HA version: [2024.3.0b0]
 - spotcast version: [3.7.1]

## Configuration

````yaml
spotcast:
  sp_dc: !secret spotcast_sp_dc
  sp_key: !secret spotcast_sp_key
````

## Logs

* normal

````log
Logger: homeassistant.setup
Source: setup.py:296
First occurred: 23:38:46 (1 occurrences)
Last logged: 23:38:46

Setup failed for custom integration 'spotcast': Unable to import component: cannot import name 'LaunchError' from 'pychromecast.error' (/usr/local/lib/python3.12/site-packages/pychromecast/error.py)
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 296, in _async_setup_component
    component = await integration.async_get_component()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 857, in async_get_component
    comp = self.get_component()
           ^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 884, in get_component
    ComponentProtocol, importlib.import_module(self.pkg_path)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/config/custom_components/spotcast/__init__.py", line 61, in <module>
    from .spotcast_controller import SpotcastController
  File "/config/custom_components/spotcast/spotcast_controller.py", line 20, in <module>
    from .spotify_controller import SpotifyController
  File "/config/custom_components/spotcast/spotify_controller.py", line 15, in <module>
    from pychromecast.error import LaunchError
ImportError: cannot import name 'LaunchError' from 'pychromecast.error' (/usr/local/lib/python3.12/site-packages/pychromecast/error.py)
````

**Comment by Daemo00** [comment]
Launcherror exception has been removed in https://github.com/home-assistant-libs/pychromecast/commit/497146909ecf79742faad5ab417c884b9dae37c9

**Comment by hmmbob** [comment]
Fix in https://github.com/fondberg/spotcast/pull/428

**Comment by FrancoLoco** [comment]
thx for fix

**Comment by FrancoLoco** [comment]
> @Daemo00 can you confirm issue is fixed with v3.7.2?

I can confirm it's working on my side

**Comment by Daemo00** [comment]
> @Daemo00 can you confirm issue is fixed with v3.7.2?

Yes no more errors with 3.7.2, thanks!
